images/buildkit/tests/02-build.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+# Build a simple image
+TMPDIR="$(mktemp -d)"
+echo "FROM scratch" > ${TMPDIR}/Dockerfile
+docker run \
+  --rm \
+  --privileged \
+  -v ${TMPDIR}:/tmp/work \
+  --entrypoint buildctl-daemonless.sh \
```

Comment:
per your TODOs about `awk` etc: Would it make more sense for us to make this the entrypoint and include `busybox` (which provides most posix utils)?

Reply:
We could include busybox / bash - I was holding off on it because I wasn't sure how minimal we wanted to make the non-dev variant. These are only needed because of how buildkit recommends launching daemonless via the script. AFAIK these aren't needed when using this as a daemon, and it's probably okay to say "if you want to use daemonless use the dev variant"? We can also wait for any feedback on this and add it later.

daemonless is not the default mode for buildkit, so I'd say it doesn't make sense to make this the entrypoint. We're using it in tests because I didn't know how we can install script dependencies in tests (since we need the buildctl CLI). Running it daemonless allows us to encapsulate everything in a single docker command.